protocol/errorsHelp.json

```
@@ -22,6 +22,7 @@
   "40112": "https://faqs.ably.com/error-code-40112-account-blocked-message-limits-exceeded",
   "40114": "https://faqs.ably.com/error-code-40114-account-wide-peak-channel-limit-exceeded",
   "40115": "https://faqs.ably.com/error-code-40115-account-restricted-request-limit-exceeded",
+  "40125": "https://faqs.ably.com/error-40125-max-number-of-rules-per-application",
```

The surrounding links are like `https://faqs.ably.com/error-code-<code>-xxx`, any reason this isn't the same? i.e.:

```suggestion
  "40125": "https://faqs.ably.com/error-code-40125-max-number-of-rules-per-application",
```